### ISSUE TITLE
needs whitelist of URLs and IPs for both `grader_endpoint` and `authentication_endpoint`

### ISSUE THREAD

**Issue by OmarIthawi** [appsembler/xblock-grade-fetcher#17]

**Comment by amirtds** [comment]
@OmarIthawi I need more details about this. It's not clear for me what we need to do. Thanks

**Comment by amirtds** [comment]
Should we use something like https://docs.python-requests.org/en/latest/user/advanced/#proxies to make request to the grader and auth endpoint ?

**Comment by OmarIthawi** [comment]
Yes @amirtds. I've written few more details in the Black Team task: https://appsembler.atlassian.net/browse/BLACK-2365
